RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1385,60 +1396,68 @@ boolean cloverUsageInit()
 	{
 		abort("Called cloverUsageInit but have no clovers");
 	}
-
+	int haveIntrinsicConstant = 2147483647; 
 	backupSetting("cloverProtectActive", false); // maybe set this before we return?
-	if(item_amount($item[Ten-Leaf Clover]) > 0)
+	//do we already have Lucky!?
+	if(have_effect($effect[Lucky!]) == haveIntrinsicConstant)
 	{
 		return true;
 	}
 
-	if(item_amount($item[Disassembled Clover]) > 0)
+	//use a clover if we have one in inventory or closet
+	if(item_amount($item[11-Leaf Clover]) < 1 && closet_amount($item[11-Leaf Clover]) > 0)
 	{
-		if(!in_glover() && !in_bhy())
-		{
-			use(1, $item[Disassembled Clover]);
-		}
+		take_closet(1, $item[11-Leaf Clover]);		
 	}
-	if(item_amount($item[Ten-Leaf Clover]) > 0)
+	if(item_amount($item[11-Leaf Clover]) > 0)
 	{
-		return true;
+		use(1, $item[11-Leaf Clover]);
 	}
-
-	if(in_koe() && spleen_left() > 1 && canChew($item[lucky pill]) && item_amount($item[rare Meat isotope]) >= 60)
+	if(have_effect($effect[Lucky!]) == haveIntrinsicConstant)
 	{
-		retrieve_item(1, $item[lucky pill]);
-		autoChew(1, $item[lucky pill]);
-		use(1, $item[Disassembled Clover]);
+		auto_log_info("Clover usage initialized");
+		return true;
 	}
-
-	if(closet_amount($item[Ten-Leaf Clover]) > 0)
+	else
 	{
-		take_closet(1, $item[Ten-Leaf Clover]);
+		auto_log_warning("Did not acquire Lucky! after using an 11-Leaf Clover");
 	}
-	if(item_amount($item[Ten-Leaf Clover]) > 0)
+
+	//use Astral Energy Drinks if we have room
+	if(spleen_left() >= 5)
 	{
-		auto_log_info("Clover usage initialized");
-		set_property("_autoCloverNext", true);
-		return true;
+		if(item_amount($item[[10883]Astral Energy Drink]) < 1 && closet_amount($item[[10883]Astral Energy Drink]) > 0)
+		{
+			take_closet(1, $item[[10883]Astral Energy Drink]);		
+		}
+		if(item_amount($item[[10883]Astral Energy Drink]) > 0)
+		{
+			use(1, $item[[10883]Astral Energy Drink]);
+		}
+		if(have_effect($effect[Lucky!]) == haveIntrinsicConstant)
+		{
+			auto_log_info("Clover usage initialized");
+			return true;
+		}
+		else
+		{
+			auto_log_warning("Did not acquire Lucky! after drinking an Astral Energy Drink");
+		}
 	}
-	abort("We tried to initialize clover usage but do not appear to have a Ten-Leaf Clover");
+
+
+	abort("We tried to initialize clover usage but was unable to get Lucky!");
 	return false;
 }
 
 boolean cloverUsageFinish()
 {
+	int haveIntrinsicConstant = 2147483647; 
 	restoreSetting("cloverProtectActive");
```

Comment:
you can remove any reference to this `cloverProtectActive` property it's not needed any more

Reply:
Wasn't clear to me what this property did. Thanks for confirming it can be removed. Removed uses of it and added it to the function which removes obsolete properties as well